edudoc/disciplinary-crash-course.md

```diff
@@ -0,0 +1,96 @@
+
+# WCA Staff Members {.page-break-before}
+
+* Like any other competitors, report any violations of the Code of Conduct of WCA Staff to the WDC. Be sure to also use ‘CC’ to include the WCA Ethics Committee (WEC) in your email.
+* If you need to report a violation of the Code of Conduct or Code of Ethics of a Staff member who is a member of the WDC or the WEC, then please report this behavior in one of the following ways:
+  * Contact the Leader and/or Senior members of the Committee usually responsible for handling the infraction.
+    * E.g. If a WDC member has violated the Code of Conduct, then contact the WDC leader and/or senior members of the WDC.
+  * Contact the WDC if a WEC member has committed a violation of the Code of Ethics.
+  * Contact the WEC if a WDC member has committed a violation of the Code of Conduct.
+
+# Cheating
+
+* If cheating is obvious, try to get evidence (especially videos, photos, names of people involved, and email addresses of witnesses) and disqualify the competitor using Regulation [2k2](https://www.worldcubeassociation.org/regulations/#2k2). At your discretion, you may let the suspect complete the attempt or the round before disqualifying them, in order to gather more evidence.
+  * For example, if a competitor is reported to have peeked on their first attempt of 3x3 blindfolded, it may be desirable to watch them/record them peeking on their 2nd and 3rd attempts before disqualification, as opposed to immediately disqualifying the competitor. This allows the WDC to have greater evidence, which is necessary for taking disciplinary action.
+  * Explain to competitors why their actions were wrong, try to explain the importance of fairness, and ask them if they have cheated in similar ways in the past. Explain to them that their actions will be reported to the WDC and that the WDC may contact them. Also explain that if a WDC investigation is started, that honesty will result in the best outcome for the competitor.
+* Tag the WDC in the Delegate report about the case. Give a rough outline of the incident for the education of other Delegates, but avoid including any evidence or identifying information in the report such as videos that reveal the identity of competitors or names of competitors accused of cheating.
+* Contact the WDC directly with a detailed description of the incident, and any evidence gathered.
+
+# Incidents Outside the Scope of the Code of Conduct
+The Code of Conduct is limited in scope to WCA Competitions and online on official WCA platforms. As a result, the WDC cannot take any action in response to incidents that happen outside of this scope. If concerning behavior happens outside of this scope, below are some recommendations.
+* Contact authority figures that may be able to resolve the situation, such as parents, school administrators, or moderators of a social media group.
```

Comment:
There needs to be a new line before a new set of bullet points, or else they just get added on to the end of the previous sentence. Annoying markdown stuff.